tests/useful/image.js

```diff
@@ -0,0 +1,30 @@
+import chai from 'chai';
+
+import fs from 'fs';
+import nconf from 'nconf';
+import nock from 'nock';
+import path from 'path';
+
+import image from '../../src/commands/useful/image';
+
+
+let sandbox;
+chai.should();
+
+const original_request_fixture = fs.readFileSync(path.join(__dirname, '../fixtures/image.json'));
+
+describe.only('image', () => {
+  before(() => {
+    nconf.set('BING_IMAGE_KEY', '123');
+  });
+  nock.cleanAll()
+  nock('https://api.cognitive.microsoft.com:443', {"encodedQueryParams":true})
+    .get('/bing/v5.0/images/search')
+    .query({"q":"cats","count":"1","offset":"0","mkt":"en-us","safeSearch":"Moderate"})
```

Comment:
I had that originally, and it didn't work either - same with having or not having 443